packages/symbiosis/src/utils.ts

```diff
@@ -0,0 +1,63 @@
+import type { ActionParams } from '@rabbitholegg/questdk'
+import type { Address, Hash } from 'viem'
+
+export enum Chains {
+  ETHEREUM = 1,
+  OPTIMISM = 10,
+  BINANCE_SMART_CHAIN = 56,
+  GNOSIS = 100,
+  POLYGON_POS = 137,
+  ZK_SYNC_ERA = 324,
+  POLYGON_ZK = 1101,
+  MANTLE = 5000,
+  BASE = 8453,
+  ARBITRUM_ONE = 42161,
+  AVALANCHE = 43114,
+  LINEA = 59144,
+  SCROLL = 534352,
+}
+
+interface Transaction {
+  chainId: number
+  from: Address
+  hash?: Hash
```

Comment:
Curious about why hash is optional here? I always thought transactions have a hash?

Reply:
I sometimes use simulated transactions if I cant find an actual transaction on the explorer.

The hash is not needed for the plugin, it is put there in case we want to find the transaction again in the explorer.